### PR TITLE
Change etcd datadir to /var/lib/etcd

### PR DESCRIPTION
**Pull request by fgimenez**
Towards #651

The dep update warned about an outdated dependency on ipam (it was used previously for e2e), I've also taken the opportunity to remove it.

### CODE REVIEW

File: service/version_bundles.go [hunk 1]
```diff
@@ -378,6 +378,11 @@ func NewVersionBundles() []versionbundle.Bundle {
 					Description: "Add AWS resource tag with GiantSwarm Cluster ID for reporting.",
 					Kind:        versionbundle.KindAdded,
 				},
+				{
+					Component:   "aws-operator",
+					Description: "Change default etcd data dir to /var/lib/etcd.",
+					Kind:        versionbundle.KindAdded,
```
Comment (rossf7) [inline service/version_bundles.go]: Nitpick, I think should be KindChanged.

File: service/awsconfig/v2/cloudconfig/templates.go [hunk 2]
```diff
@@ -210,7 +210,7 @@ Before=set-ownership-etcd-data-dir.service etcd3.service
 
 [Mount]
 What=/dev/xvdh
-Where=/etc/kubernetes/data/etcd
+Where=/var/lib/etcd
```
Comment (xh3b4sd) [inline service/awsconfig/v2/cloudconfig/templates.go]: Why is this? I think v2 is frozen and should not change. The same change happened for v5 and we have a changelog for the corresponding version bundle. There is no changelog for the version bundle of the corresponding changes for v2 though. This feels super wrong. I am not sure about the implications here. I also think that it would be good to have separate PRs for changes on different versions.

Comment (kopiczko) [reply service/awsconfig/v2/cloudconfig/templates.go]: Eagle eye Tim :) +1 I think it affects only `v_3_1_1` which shouldn't be imported in `v2`.

Comment (fgimenez) [reply service/awsconfig/v2/cloudconfig/templates.go]: Indeed, very good catch, this is not needed at all, the only required template change is in `/service/awsconfig/v5/templates/cloudconfig/mount_etcd_volume.go` probably something went wrong with my rebase after the latest template changes, removed.
